<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TermDepositOperation</class>
 <widget class="QWidget" name="TermDepositOperation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="layout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="note_label">
     <property name="text">
      <string>Note</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="main_label">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Term Deposit</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="vertical_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="11">
    <widget class="QPushButton" name="revert_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Cancel changes</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="7">
    <widget class="QPushButton" name="del_button">
     <property name="toolTip">
      <string>Remove activity</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="8">
    <widget class="TableViewWithFooter" name="actions_table">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="details_label">
     <property name="text">
      <string>Deposit activity</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="account_label">
     <property name="text">
      <string>Account</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QPushButton" name="commit_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Commit changes</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="add_button">
     <property name="toolTip">
      <string>Add activity</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2" colspan="6">
    <widget class="QLineEdit" name="note"/>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="AccountSelector" name="account_widget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableViewWithFooter</class>
   <extends>QTableView</extends>
   <header>jal/widgets/custom/tableview_with_footer.h</header>
  </customwidget>
  <customwidget>
   <class>AccountSelector</class>
   <extends>QWidget</extends>
   <header>jal/widgets/reference_selector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
